certification/internal/shell/podman.go

```diff
@@ -136,3 +136,89 @@ func (pe PodmanCLIEngine) ScanImage(image string) (*cli.ImageScanReport, error)
 
 	return &cli.ImageScanReport{Stdout: out.String(), Stderr: stderr.String()}, nil
 }
+
+// Create simply creates a stopped container from the image provided and returns the container ID.
+// It is the responsibility of the caller to clean up the image after use.
+func (pe PodmanCLIEngine) Create(rawImage string, opts *cli.PodmanCreateOptions) (*cli.PodmanCreateReport, error) {
+	cmdArgs := []string{"create"}
+
+	if opts.Entrypoint != "" {
+		cmdArgs = append(cmdArgs, "--entrypoint", opts.Entrypoint)
+	}
+
+	cmdArgs = append(cmdArgs, rawImage)
+
+	cmd := exec.Command("podman", cmdArgs...)
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	log.Trace("Running podman with the following invocation", cmdArgs)
+	err := cmd.Run()
+	if err != nil {
+		return &cli.PodmanCreateReport{
+			Stdout: stdout.String(),
+			Stderr: stderr.String(),
+		}, nil
+	}
+
+	return &cli.PodmanCreateReport{
+		Stdout:      stdout.String(),
+		Stderr:      stderr.String(),
+		ContainerID: strings.TrimSuffix(stdout.String(), "\n"),
```

Comment:
Nit: TrimSpace seems to work better here. It encompasses more whitespace, and is more platform independent.

Reply:
Fixed!